src/translations/translationHelpers.ts

```diff
@@ -0,0 +1,27 @@
+import LANGS from './constants'
+
+function getCurrentLanguageValue(){
```

Comment:
getCurrentLanguageValue() { - missed space before '{'

Please, leave a comment for this functions to understand what it does. For this purpose you can use JSDoc format:

/**
 * Returns language code in terms of current pathname in the URL
 * @example
 * // returns default code, e.g. 'en'
 * href: http://the-lessoner/home
 * @example
 * // returns 'ru'
 * href: http://the-lessoner/ru/level1/level2
 * @returns {String}
 */
function getCurrentLanguageValue() {